Sources/rswift/main.swift

```diff
@@ -142,4 +142,33 @@ command(
 
   try RswiftCore.run(callInformation)
 
-}.run(Rswift.version)
+}
+
+// Temporary warning message during migration to R.swift 4
+let parser = ArgumentParser(arguments: CommandLine.arguments)
+_ = parser.shift()
+let exception = parser.hasOption("version") || parser.hasOption("help")
+
+if !exception && parser.shift() != "generate" {
+  var arguments = CommandLine.arguments
+  arguments.insert("generate", at: 1)
+  let command = arguments
+    .map { $0.contains(" ") ? "\"\($0)\"" : $0 }
+    .joined(separator: " ")
+
+  let message = "⚠️ R.swift 4 requires \"generate\" command as first argument to the executable.\n"
```

Comment:
Does Xcode pick this up as a build error? I think the build will fail due to the `exit(EXIT_FAILURE)`, but I'm not sure if the `fputs("\(message)\n", stderr)` will be shown as a build error.

In other places we print `error: message` to the console to get Xcode to show it.